src/runvm-osbuild

```diff
@@ -60,7 +60,7 @@ extra_kargs=$(getconfig "extra-kargs-string" "")
 # set the OSTree repo and ref information
 ostree_ref=""
 ostree_repo=""
-if [ -z "${deploy_via_container}" ]; then
+if [ -n "${deploy_via_container}" ]; then
```

Comment:
Hmm, I don't think that's right. If we're deploying via container, then we don't need the `ostree_ref` and `ostree_repo` variables. If we're not, then it means we're pulling from an OSTree repo so we do.

The Prow failure in CI is catching this (because RHCOS uses `deploy-via-container: true` but FCOS yet).